node_test/rpc_test.go

```diff
@@ -403,6 +404,11 @@ func setupNitroNodeWithRPCClient(
 		t.Fatal(err)
 	}
 
+	cert, err := tls.LoadX509KeyPair("../internal/tls/statechannels.org.pem", "../internal/tls/statechannels.org_key.pem")
```

Comment:
I think relative paths are safe to use in a test file. This seems to function correctly if I run `go test ./...` from the root directory and if I run it from the `node_test` directory.